ssdb-configuration-options.adoc

```diff
@@ -1,4 +1,10 @@
-= SSDB Configuration Options
+= SSDB Installation and Configuration
+
+== Installation
+
+{product-name} packages do not include SSDB server and {product-name} installation and configuration utilities do not alter SSDB configuration. To install SSDB, follow instructions provided by SSDB developer community in http://ssdb.io/docs/install.html[SSDB Installation Documentation].
```

Comment:
Those instructions are for getting the latest SSDB.  Fine as far as it goes but I think we should be clear on which version we tested with and perhaps provide an alternative installation method to get the specific version we tested - e.g. based on the script (old now perhaps 1.9.3 is no longer appropriate):
```
#!/bin/bash
SSDB_INSTALL_DIR="/home/zimbra/ssdb"
if [ -d "${SSDB_INSTALL_DIR}" ]
then
    echo "### Assuming ssdb is already setup under ${SSDB_INSTALL_DIR}"
else
    export SSDBVERSION=1.9.3
    export SSDBDIR="${SSDB_INSTALL_DIR}"
    if [ -d ${SSDBDIR} ]
    then
        echo "${SSDBDIR} already exists!"
        exit 1
    fi
    (
        set -x
        sudo apt-get install autoconf
        mkdir -p ${SSDBDIR} &&
        cd ${SSDBDIR} &&
        git clone -n --depth=1 -b stable-${SSDBVERSION} -o origin https://github.com/ideawu/ssdb.git &&
        cd ssdb &&
        git fetch --all &&
        git checkout stable-${SSDBVERSION} &&
        make PREFIX=${SSDBDIR} && make install PREFIX=${SSDBDIR} &&
        ${SSDBDIR}/ssdb-server -d ${SSDBDIR}/ssdb.conf
        ps -ef | grep ssdb-server
        ${SSDBDIR}/ssdb-server -d ${SSDBDIR}/ssdb.conf -s stop
    )
fi
```